src/main/java/de/hysky/skyblocker/skyblock/slayers/SlayerEntitiesGlow.java

```diff
@@ -80,13 +87,21 @@ public static Box getSlayerMobBoundingBox(LivingEntity entity) {
      * @param armorStand  the entity that contains the display name of the Slayer (mini)boss
      */
     private static MobEntity findClosestMobEntity(Class<? extends MobEntity> entityClass, ArmorStandEntity armorStand) {
-        return armorStand.getWorld().getEntitiesByClass(entityClass, armorStand.getDimensions(null)
-                        .getBoxAt(armorStand.getPos()).expand(1.5), entity ->
-                        !entity.isDead() && entity.age > armorStand.age - 4 && entity.age < armorStand.age + 4)
+        List<MobEntity> mobEntities = armorStand.getWorld().getEntitiesByClass(entityClass, armorStand.getDimensions(null)
+                        .getBoxAt(armorStand.getPos()).expand(0.3f, 1.5f, 0.3f), entity -> !entity.isDead())
                 .stream()
                 .filter(SlayerEntitiesGlow::isValidSlayerMob)
-                .min(Comparator.comparingDouble((MobEntity e) -> e.distanceTo(armorStand)))
-                .orElse(null);
+                .sorted(Comparator.comparingDouble((MobEntity e) -> e.distanceTo(armorStand)))
```

Comment:
Since you seem to like optimizing, you can do `squaredDistanceTo`.
```suggestion
                .sorted(Comparator.comparingDouble(e -> e.squaredDistanceTo(armorStand)))
```